releases.js

```diff
@@ -0,0 +1,40 @@
+const PLATFORM_WEB = 'Web';
+const PLATFORM_SERVER = 'Server';
+const PLATFORM_IOS = 'iOS';
+const PLATFORM_ANDROID = 'Android';
+const PLATFORM_REACT_NATIVE = 'React Native';
+const PLATFORM_FLUTTER = 'Flutter';
+
+
+export const releases = [
+    {
```

Comment:
why is this not driven via some sort of api?

Reply:
Will be updated by a script at build time in the next release